content/docs/plugins/dot-reporter.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Dot reporter
+description: Minimalist reporter for Japa
+---
+
+# Dot reporter
+
+Reporters in Japa are responsible for displaying/saving the progress of tests. The dot reporter is a minimalist reporter that displays a dot for each passing test and an cross for each failing test.
+
+![](https://user-images.githubusercontent.com/8337858/188954371-959f6171-22c8-4a15-a76f-b9d9c2576ab6.png)
```

Comment:
I think you can upload the image in the same repo and reference that. Because, after a while we will forget that we are using the repo image as a reference and if we change the README, we will end up breaking the docs